pom.xml

```diff
@@ -982,6 +982,11 @@
 				<artifactId>jackson-databind</artifactId>
 				<version>${jackson.version}</version>
 			</dependency>
+            <dependency>
```

Comment:
In this and all other XML files I notice a different style of indentation. Could we correct that please?